blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/FavoriteContainerImpl.java

```diff
@@ -41,6 +42,9 @@ public BlueFavorite get(String name) {
         Jenkins j = Jenkins.getInstance();
 
         for(final Item favorite: Favorites.getFavorites(user.user)){
```

Comment:
If you move the access of `Favorites.getFavorites` into `FavouriteUtil` and have the filtering happen there, you could write a test for it.